feature/google/src/main/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -15,10 +16,15 @@ fun ProductDetails.toInAppStoreProduct(): StoreProduct? = this.toStoreProduct(em
 fun ProductDetails.toStoreProduct(
     offerDetails: List<ProductDetails.SubscriptionOfferDetails>
 ): GoogleStoreProduct? {
-    val subscriptionOptions = offerDetails.map { it.toSubscriptionOption(productId, this) }
-    val defaultOffer = subscriptionOptions.findDefaultOffer()
+    val subscriptionOptions = if (productType.toRevenueCatProductType() == ProductType.SUBS) {
```

Comment:
😮 

i guess this worked before because offerDetails would be null for an OTP?

Reply:
TBH... I'm not quite sure lol

I _might_ have been empty before 😬